owlplug-client/src/main/resources/owlplug.css

```
@@ -50,15 +48,8 @@
     -fx-font-family: "Segoe UI";
 
-    disabled-color: -color-fg-muted;
-    danger-color: -color-danger-emphasis;
-    warning-color: -color-warning-emphasis;
-    success-color: -color-success-emphasis;
-
-    theme-primary-color: -color-accent-fg;
-
-    text-color: -color-fg-default;
-    text-emphase-color: -color-fg-emphasis;
+    -owlplug-color-text: -color-fg-default;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)